docs/en/engines/table-engines/index.md

```diff
@@ -79,6 +79,16 @@ Engines in the family:
 - [Buffer](../../engines/table-engines/special/buffer.md#buffer)
 - [KeeperMap](../../engines/table-engines/special/keepermap.md)
 
+### Engine summary
+
+The following comparison table is not meant to be exhaustive.
+
+| Features                  | MergeTree          | ReplicatedMergeTree | Log             | Memory          | KeeperMap         | RocksDB          |
+|---------------------------|--------------------|---------------------|-----------------|-----------------|-------------------|------------------|
+| Replication               | Local              | Replicated          | Local           | Local           | Replicated        | Local            |
```

Comment:
MergeTree, Log, Memory and RocksDB don't have "replication" (in the sense of data replicate across cluster nodes) at all.